hawc/apps/animal/exports.py

```diff
@@ -167,6 +168,8 @@ def _get_dose_units(cls, doses):
 
     @classmethod
     def _get_doses_str(cls, doses):
```

Comment:
would you mind adding annotations here pls? since you just touched it and it's a tiny little function, it'd be easy to add. Thanks